includes/api/class-wc-rest-network-orders-controller.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * REST API Network Orders controller
+ *
+ * Handles requests to the /orders/network endpoint
+ *
+ * @author   WooThemes
+ * @category API
+ * @package  WooCommerce/API
+ * @since    @todo
```

Comment:
@claudiulodro 3.3 or 3.4?

Reply:
3.3 is good. This just has small changes left.